build/rubygem-simp-cli.spec

```diff
@@ -1,3 +1,45 @@
+%{lua:
+--
+-- When buildng the RPM, declare macro 'pup_module_info_dir' with the path to
```

Comment:
* Why use `pup_module_info_dir`? `component_info_dir` would be more generic
* Typo:  `buildng` -> `building`

Reply:
It's the macro set by pkg:single.
It was started by the pupmods and later used by simp-doc.